src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -72,11 +72,11 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
                 continue;
             }
 
-            String pathAdjusted = file.absolutePath().replace('\\', '/');
+            String pathAdjusted = file.absolutePath().replace('\\', '/').toLowerCase();
```

Comment:
What's the deal here? If you're on Linux with two files that differ only by case then the plugin'll blow up.